### DISCUSSION TITLE
Been Busy?!

### DISCUSSION THREAD

**Discussion by corygalyna** [ZORALab/Hestia#417] [Q&A] [answered]
@JeanShuralyov @hollowaykeanho , you 2 have been busy on this project again behind my back didn't you? =)

**Comment by JeanShuralyov** [comment]
Hehe. Well, gonna make it clear and nice for the search. =p

**Comment by corygalyna** [comment]
really? Could have just call me in then.

**Comment by JeanShuralyov** [comment]
Nahh, big guy has been busy touching up secretly. @hollowaykeanho ?

**Comment by corygalyna** [comment]
I see no differences but I do see some progress in the experimental branch. What's the update?

**Comment by JeanShuralyov** [comment, marked as answer]
Mostly marketing stuffs like rewriting the landing pages, touching up the filesystem, and etc. Next thing is README.md and GitHub platform related matters.

**Comment by corygalyna** [comment]
@hollowaykeanho , are you there? Where did he go?!

**Comment by JeanShuralyov** [comment]
Busy.. I guess.. you know him more than I do.

**Comment by corygalyna** [comment]
Ok ok.. I will contact him privately.
